ecosystem/indexer-grpc/indexer-transaction-generator/src/config.rs

```diff
@@ -101,6 +125,31 @@ pub struct TransactionImporterPerNetworkConfig {
     pub versions_to_import: HashMap<u64, String>,
 }
 
+/// Configuration for generating transactions from scripts.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ScriptTransactionGeneratorConfig {
+    /// List of scripts to run to generate transactions.
+    /// Note, each run will be executed in sequence.
+    pub runs: Vec<ScriptTransactions>,
```

Comment:
nit: maybe rename this field to transactions to be more intuitive? wdyt?

Reply:
transactions is the subfield of runs; it'll degrade the readabilit.